code-challenges.test.js

```diff
@@ -11,22 +11,81 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+//input: lenght of sequence
+// output: sequence of specified lenght 
+// method: we use a conditinal statement to evaluate if the number is equal to two, if so we define what the outcome would be. In our else statement we tell it to look at the last number befroer the new number we just recieved in the sequence, and tell it to push the two last alues and add them together to recieve our new value.
+
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("fibSeq", () => {
+  it("returns the last two numbers added together to create a seq", ()=> {
+    expect(fibSeq(fibLength1).toEqual([1, 1, 2, 3, 5, 8]))
+    expect(fibSeq(fibLength2).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]))
+  })
+
+})
+
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+const newNum = fibLength1[fibLength1.length - 1] + (fibLength1[fibLength1.length - 2] || 0);
+
+const fibSeq = (number) => {
+  if(number===2)  {
+    return [1,1]
+  }
+  else  {
+    let sequence = fibSeq (number -1)
+    sequence.push(sequence[sequence.length -1] + sequence[sequence.length - 2])
+    return sequence
+  }
+}
+console.log(fibSeq(fibLength1));
+
+
+//Tried †o make this code work but needed assitance from student to fix my sequence
+//   for (let i = 0; i < fibLength; i++) {
+//     fibLength.push(num1);
+//     nextNum = num1 + num2;
+//     num1 = num2;
+//     num2 = nextNum;
+    
+//   }
+  
+
+//   return  (
+//     fibLength
+//   )
+
+// };
+
+// console.log(sequence);
+
+
+
+
 
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
+//input: objects
+//output: sorted numerical vals in objects
+// Method: use a function that evaluates the value of the object and using .sort we tell it to sort our array into the direction we want it to and show the results
```

Comment:
Same recommendation as above. If you break each step down and are able to fully understand what's going on, you will then be able to write much better pseudo code that describes what your code is doing.